src/background.js

```diff
@@ -38,8 +36,13 @@ function handleToolsMessage(msg, port) {
   }
 }
 
-function handlePageMessage(msg, port) {
-  const tools = toolsPorts.get(port.sender.tab.id)
+// Receive messages from content scripts
+chrome.runtime.onMessage.addListener((msg, sender) => {
+  handlePageMessage(msg, sender.tab.id)
+});
```

Comment:
I would prefer that single statement lambda functions don't use braces.

Reply:
Removed!